src/services/profiles-service.js

```diff
@@ -5,7 +5,7 @@ import axios from "axios";
 // change this to point to your server on Heroku
 
 // const BASE_URL = "https://engineer-software-dat-a3.herokuapp.com";
-const BASE_URL = "http://localhost:5000";
+const BASE_URL = "http://localhost:5002";
```

Comment:
Any reason why you're changing it to local host:5002?

Reply:
IF there's no good reason please make a change back to 5000 as we don't want to switch this around too much.

Reply:
port changed to 5000